homeassistant/components/bond/light.py

```diff
@@ -119,6 +122,8 @@ def is_on(self) -> bool:
 
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn the fireplace on."""
+        _LOGGER.debug("fireplace async_turn_on called with: %s", kwargs)
```

Comment:
We start logging messages with capital letter. The service call is already logged by the core. Is it the fireplace word that is interesting?

Reply:
@prystupa can you check the service call log and see if it makes this unnecessary? Otherwise we will want to fix the capitalization of this

Reply:
will do